### PR TITLE
Add more Portuguese translations

### PR DESCRIPTION
**Pull request by kaykessantos**

### CODE REVIEW

File: src/locales/translations/pt.ts [hunk 1]
```diff
@@ -69,9 +69,9 @@ export default {
   update_modal_remind_me_next_time: "Me lembre mais tarde",
   update_modal_skip_this_update: "Ignorar esta atualização",
   gta_path_modal_cant_find_game_title:
-    "Não foi possível encontrar o GTA San Andreas!",
+    "Não foi possível encontrar o GTA: San Andreas!",
```
Comment (Southclaws) [inline src/locales/translations/pt.ts:72]: I'm not sure the colon is necessary - is this used in other copy? I've only ever seen it written as "GTA San Andreas"

Comment (pushline) [reply src/locales/translations/pt.ts:72]: This colon is not necessary, i forgot to remove on one of my updates to pt translation

Comment (kaykessantos) [reply src/locales/translations/pt.ts:72]: Regarding the colon, I asked why in the English language it contained the colon in the rest of the words, I also researched it and found it in the original name

https://prnt.sc/Sd4_hK9ZhcF8

**Comment by Southclaws** [comment]
If it's different enough, but it's worth bearing in mind that we don't have any fallback aliases for language groups.

What this means is, if a user lands on a pt-BR page from Google and it's not translated, but the pt-PT version is, that user has no way to know that because they would be served the en-GB page as a fallback.

There are also a lot of documentation pages and I don't think _any_ language has achieved 100% translation of every single page yet. It's a huge task to keep up to date and could be easier to just stick with pt-BR as the grouping but keep the language as general enough (no slang, etc) that it could be understood by as many people as possible - regardless of dialect.

**Comment by Southclaws** [comment]
okay I'm an idiot this is the launcher not the website, do a pt-PT version!

**Pull request merged**